### PR TITLE
Introduce commands to generate tests for function at cursor

### PR DESCRIPTION
**Pull request by krobelus**

### CODE REVIEW

**Comment by zimmski** [comment]
Usage of the README needs to be adapted, since we now allow unit test skeletons as well

File: README.md [hunk 1]
```diff
@@ -27,9 +27,12 @@ This plugin integrates [Symflower CLI](https://symflower.com/en/products/symflow
 
 - The `symflower` command generates tests for source files in the current directory.
 - The `symflower-alternate-file` command jumps between a source file and the generated test file.
+- The `symflower-buffile` command generates tests for the current buffer's file.
```
Comment (zimmski) [inline README.md]: Usage of the README needs to be adapted, since we now allow unit test skeletons as well

File: rc/symflower.kak [hunk 2]
```diff
@@ -49,3 +75,10 @@ define-command symflower-alternative-file -docstring 'Jump to the alternate file
 	esac
 	printf "edit -- '%s'" "$(printf %s "$altfile" | sed "s/'/''/g")"
 }}
+
+declare-user-mode symflower
+map global symflower a %{: symflower-alternative-file<ret>} -docstring "implementation ←→ Symflower test"
```
Comment (zimmski) [inline rc/symflower.kak:92]: Can you please add a feature section to the README to showcase both the unit test generation and unit test skeleton adding with a GIF and text for the user on what these features are and how to use them

**Review by zimmski — CHANGES_REQUESTED** [review]

File: rc/symflower.kak (hunk 2 above)
Comment (krobelus) [reply rc/symflower.kak:92]: Added screencasts but no separate feature section yet

**Pull request merged**